Makefile

```diff
@@ -0,0 +1,10 @@
+.PHONY: install-dependencies doc/generate dev
+
+install-dependencies:
+	yarn install
```

Comment:
Please help to check the indent. ✨ (using tab with 2 spaces instead)
```suggestion
  yarn install
```

Reply:
@sanG-github For Makefile, `TAB` is required

Reply:
Ah okay, it shows `TAB with 2 spaces` in my local, but seems like it shows `TAB with 8 spaces` here 😢 

| Local                                                                                                                                   	| Remote                                                                                                                                  	|
|-----------------------------------------------------------------------------------------------------------------------------------------	|-----------------------------------------------------------------------------------------------------------------------------------------	|
| <img width="385" alt="image" src="https://github.com/nimblehq/api-mock-templates/assets/63148598/628307cf-2f76-4c5a-bdf1-859598b8bb0b"> 	| <img width="483" alt="image" src="https://github.com/nimblehq/api-mock-templates/assets/63148598/e1029b6b-b8e2-4f37-97ef-e62f66c1c5e8"> 	|